### PR TITLE
Simplify the logging system

### PR DESCRIPTION
**Pull request by ozxybox**
I tried to clean up the logging system a bit to be a little less complicated. Note: I can't test the UI changes on my machine atm.

I kept getting confused as to why the format option changed the logging output and not the file format. I then realized that logging was used for IPC into the UI. I tried to keep everything the same for the most part.

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -143,18 +93,16 @@ auto main( int argc, char* argv[] ) -> int {
 		excludes.emplace_back( ".*\\.vmx" );
 		excludes.emplace_back( ".*\\.log" );
 		excludes.emplace_back( ".*verifier_index\\.rsv" );
-		ret = create( root, indexLocation, excludes, overwrite, output.get() );
+		ret = create( root, indexLocation, excludes, overwrite );
 	} else {
 		// warn about stuff which shouldn't be here, don't use the `output::report` as this is a negligible user error
```
Comment (ENDERZOMBI102) [inline src/main.cpp]: i'd update the comment too

Comment (ozxybox) [reply src/main.cpp]: fixed it!

**Review by ENDERZOMBI102 — APPROVED** [review]

File: src/main.cpp (hunk 1 above)
Comment (ENDERZOMBI102) [reply src/main.cpp]: its still the same?

Comment (ozxybox) [reply src/main.cpp]: looks correct on my screen? refresh the page?

Comment (ENDERZOMBI102) [reply src/main.cpp]: oh yeah, the ui didn't update, my bad

**Pull request merged**